Hazel/src/Hazel/Core/Window.cpp

```diff
@@ -1,20 +1,24 @@
 #include "hzpch.h"
 #include "Hazel/Core/Window.h"
 
-#ifdef HZ_PLATFORM_WINDOWS
+#if defined(HZ_PLATFORM_WINDOWS)
+	#include "Platform/Windows/WindowsWindow.h"
+#elif defined(HZ_PLATFORM_LINUX)
 	#include "Platform/Windows/WindowsWindow.h"
 #endif
 
 namespace Hazel
 {
 	Scope<Window> Window::Create(const WindowProps& props)
 	{
-	#ifdef HZ_PLATFORM_WINDOWS
+	#if defined(HZ_PLATFORM_WINDOWS)
```

Comment:
Preprocessor check is unnecessary as above and only causes confusion.

Reply:
Same responds as above